app/main/index.ts

```diff
@@ -185,15 +185,18 @@ app.on('ready', () => {
 	const page = mainWindow.webContents;
 
 	page.on('dom-ready', () => {
+		// On dom-ready, fetch user agent and send to renderer processes.
+		// As of now, only system-util.ts catches it.
 		if (ConfigUtil.getConfigItem('startMinimized')) {
 			mainWindow.hide();
 		} else {
 			mainWindow.show();
 		}
-		if (!ConfigUtil.isConfigItemExists('userAgent')) {
-			const userAgent = session.fromPartition('webview:persistsession').getUserAgent();
-			ConfigUtil.setConfigItem('userAgent', userAgent);
-		}
+	});
+
+	ipcMain.handle('fetch-user-agent', async () => {
+		const userAgent = session.fromPartition('webview:persistsession').getUserAgent();
+		return userAgent;
```

Comment:
Sorry, another nitpick, would you mind just returning the result of the expression directly?
I don't see the need to declare a `const` here.

Reply:
Sure